azure-devops/azext_devops/test/repos/test_ref.py

```diff
@@ -18,15 +18,9 @@
 from azext_devops.test.utils.helper import get_client_mock_helper, TEST_DEVOPS_ORG_URL
 
 
-class TestRefMethods(unittest.TestCase):
+class AuthenticatedTests(unittest.TestCase):
```

Comment:
You can move this outside for consuming it in other tests. 👍

Reply:
Will do. I just wanted to get approval before extracting it.